src/input/SourceMouse.java

```diff
@@ -0,0 +1,90 @@
+package input;
+
+import java.nio.DoubleBuffer;
+
+import org.joml.Vector2f;
+import org.lwjgl.BufferUtils;
+import org.lwjgl.glfw.GLFW;
+
+import bus.MessageBus;
+import render.Display;
+
+public class SourceMouse implements InputSourceI {
+
+	private boolean dointeract = false;
+	private Vector2f prevMousePos = new Vector2f();
+
+	private final Display display;
+	private final MessageBus bus;
+
+	public SourceMouse(Display disp, MessageBus bus) {
+		this.display = disp;
+		this.bus = bus;
+	}
+
+	@Override
+	public boolean closeGame() {
```

Comment:
Idea: make this unnecessary by using Java 8's default interface implementation feature